starboard_api/api.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard_api/api.h"
+
+#include "starboard_api/stats_tracker.h"
+
+namespace starboard_api {
+
+int FileWrite(SbFile file, const char* data, int size) {
+  int result = SbFileWrite(file, data, size);
+  auto* stats_tracker = StatsTrackerContainer::GetInstance()->stats_tracker();
+  if (result > -1) {
+    stats_tracker->FileWriteSuccess();
+  }
+  if (result < 0) {
+    stats_tracker->FileWriteFail();
+  }
+  if (result > 0) {
+    stats_tracker->FileWriteBytesWritten(result);
```

Comment:
When```result == 0```, this function just do nothing by adding 0. So it can be merged with the case ```(result > -1)```?